Tests/UploadTests.swift

```diff
@@ -518,6 +518,32 @@ class UploadMultipartFormDataTestCase: BaseTestCase {
             XCTFail("Content-Type header value should not be nil")
         }
     }
+    
+    func testThatUploadingMultipartFormDataWithNotExistFileShouldFail() {
+        // Given
+        let urlString = "https://httpbin.org/post"
```

Comment:
```suggestion
        let urlString = URL.makeHTTPBinURL(path: "post")
```

We have APIs for generating our URLs which should be used instead of raw `String`s.